### PR TITLE
[dashboard] Update workspaces, settings, and header layout

### PR DESCRIPTION
**Pull request by gtsiolis**
More pixel shifting. 💄 

Resolves some of the issues in https://github.com/gitpod-io/gitpod/issues/3507.

| BEFORE | AFTER | 
|-|-|
| image | image |
| image  | image |
| image | image |
| image | image |

### CODE REVIEW

File: components/dashboard/src/workspaces/Workspaces.tsx [hunk 1]
```diff
@@ -127,28 +126,24 @@ export class Workspaces extends React.Component<WorkspacesProps, WorkspacesState
                             <div className="flex flex-col items-center w-96 m-auto">
                                 <h3 className="text-center pb-3">No Active Workspaces</h3>
                                 <div className="text-center pb-6 text-gray-500">Prefix a git repository URL with gitpod.io/# or open a workspace template. <a className="text-gray-400 underline underline-thickness-thin underline-offset-small hover:text-gray-600" href="https://www.gitpod.io/docs/getting-started/">Learn how to get started</a></div>
-                                <button onClick={toggleTemplateModal}>Select Template</button>
+                                <button onClick={toggleTemplateModal} className="font-medium">Select Template</button>
```
Comment (gtsiolis) [inline components/dashboard/src/workspaces/Workspaces.tsx:129]: **issue**: This triggers an empty modal for users that only authorized with a GL provider.

File: components/dashboard/src/settings/Account.tsx [hunk 2]
```diff
@@ -12,15 +12,18 @@ export default function Account() {
...
+            </div>
+            <div className="flex justify-end mt-6">
                 <button className="border-red-900 bg-red-500 hover:bg-red-700" onClick={close}>Delete Account</button>
             </div>
         </Modal>
 
-        <SettingsPage title='Account' subtitle='Profile details'>
+        <SettingsPage title='Account' subtitle='Manage account and git configuration.'>
             <h3>Profile</h3>
+            <p className="text-base text-gray-500 pb-4">The following information will be used to set up git configuration. You can override git author name and email per project by using the default environment variables <span className="text-gitpod-kumquat-dark bg-gitpod-kumquat-light px-1.5 py-1 rounded-md text-sm font-mono">GIT_AUTHOR_NAME</span> and <span className="text-gitpod-kumquat-dark bg-gitpod-kumquat-light px-1.5 py-1 rounded-md text-sm font-mono">GIT_COMMITTER_EMAIL</span>.</p>
```
Comment (gtsiolis) [inline components/dashboard/src/settings/Account.tsx:26]: **issue**: Name still is not picking up name when the user has only authorized with a GL provider.

File: components/dashboard/src/workspaces/Workspaces.tsx (hunk 1 above)
Comment (gtsiolis) [inline components/dashboard/src/workspaces/Workspaces.tsx:129]: **thought**: Font weight is not picked up when using `@apply` in `index.css`. Maybe this is by design?

**Review by svenefftinge — APPROVED** [review]
LGTM. Please squash to one commit. 🙏

File: components/dashboard/src/workspaces/Workspaces.tsx (hunk 1 above)
Comment (gtsiolis) [reply components/dashboard/src/workspaces/Workspaces.tsx:129]: Resolving, added follow up issue in https://github.com/gitpod-io/gitpod/issues/3531.

File: components/dashboard/src/settings/Account.tsx (hunk 2 above)
Comment (gtsiolis) [reply components/dashboard/src/settings/Account.tsx:26]: Resolving, added follow up issue in https://github.com/gitpod-io/gitpod/issues/3532.

**Pull request merged**